pkg/v1alpha2/registry/consumer/registery.go

```diff
@@ -0,0 +1,59 @@
+package consumer
```

Comment:
nit: typo in file name.

Reply:
sorry did not get this, the package is consumer which has multiple files with implementation, How did you want to change it?

Reply:
what Venil is saying is that there is a typo in the file name: regisTERY. When it should be regisTRY

Reply:
ok got it.